src/cljs/nr/lobby.cljs

```diff
@@ -84,10 +85,14 @@
                                "error" {:time-out 0 :close-button true}))))))))
 
 (defn leave-game []
-  (ws/ws-send! [:game/leave {:gameid (current-gameid app-state)}]
-               8000
-               #(when (sente/cb-success? %)
-                  (leave-game!))))
+  (if (= "local-replay" (:gameid @app-state))
+    (do
+      (swap! app-state assoc :gameid nil)
```

Comment:
We shouldn't be tracking :gameid on app-state anymore, which is why there's the current-gameid function. This line might not be "bad" but it's not necessary.

Reply:
It seems like we still use `:gameid` at the top level of the state in a few places. I agree that we should pick one and stick with it. Looking at all the various flows I can imagine how we aren't setting things correctly in all cases and sending people to fake games and lobbies.

Reply:
Ah geez, really? Damn, sorry about that. One of these days I'll find time and look at that mess. I thought caught everything.